containers/azure-functions-pwsh-7/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure Functions & PowerShell 7
+
+## Summary
+
+*Develop Azure Functions in PowerShell. Includes .NET Core 3.1, PowerShell 7, the Azure Functions SDK, and related extensions and dependencies.*
```

Comment:
You'll need to change this to ".NET Core, PowerShell, the Azure Functions SDK...."

Reply:
Done